images/win/scripts/Installers/Install-Selenium.ps1

```diff
@@ -21,15 +23,11 @@ Write-Host "Downloading selenium-server-standalone v$seleniumVersion..."
 
 $seleniumReleaseUrl = "https://selenium-release.storage.googleapis.com/$($seleniumVersion.ToString(2))/selenium-server-standalone-$($seleniumVersion.ToString(3)).jar"
 New-Item -ItemType directory -Path "C:\selenium\"
```

Comment:
nitpick: Duplicate the same variables name:
New-Item -ItemType directory -Path "C:\selenium\"
Start-DownloadWithRetry -Url $seleniumReleaseUrl -Name "selenium-server-standalone.jar" -DownloadPath "C:\selenium\"
$seleniumBinPath = "C:\selenium\selenium-server-standalone.jar"

C:\selenium - 3
selenium-server-standalone.jar - 2

Reply:
Refactored